internal/pkg/dependencies/abstract.go

```diff
@@ -62,6 +62,8 @@ type CommonDeps interface {
 	Template(replacements replacekeys.Keys) (*template.Template, error)
 	TemplateState(loadOptions loadState.OptionsWithFilter, replacements replacekeys.Keys) (*template.State, error)
 	TemplateDir() (filesystem.Fs, error)
+	TemplateSrcDir() (filesystem.Fs, error)
+	TemplateTestsDir() (filesystem.Fs, error)
```

Comment:
V sablone som vytvoril `src` s `tests` dir.
Napr. `my-template/v1/{src/tests}`.
A existujuce subory som presunul do `tests`.

Je to priprava na to, ze pridame moznost pisat pre sablony testy.
A aby sme to potom nemuseli migrovat, tak som to nachystal uz teraz.